docs/incremental_backups.md

```diff
@@ -64,3 +64,23 @@ For example, with a value of 2, the first two backups will be a key and a delta,
 This is important because on rare occasions a backup can be corrupted, and in the case of incremetnal backups, this corruption might impact all the following backups in the chain. Occasionally performing a full backup limits how far a corrupted delta backup can propagate.
 
 The value to use depends on your storage constraints and the frequency of your backups, but a value of 20 is a good start.
+
+## NBD-enabled Backups
+
+You have the option to utilize the NBD network protocol for data transfer instead of the VHD handler generated by the XAPI. NBD-enabled backups generally show improved speed as the load on the Dom0 is reduced.
+
+To enable NBD on the pool network, select the relevant pool, and navigate to the Network tab to modify the parameter:
+
+![](./assets/nbdconnection.png)
+
+When creating or editing a delta backup on this pool in the future, you can activate NBD under the Advanced settings:
```

Comment:
we're trying to transition to the wording incremental backup , so maybe we can use 
```suggestion
When creating or editing a incremental (formerly delta) backup and replication on this pool in the future, you can activate NBD under the Advanced settings:
```